packages/account/src/Components/poi/status/unsupported/unsupported.tsx

```diff
@@ -91,19 +89,12 @@ const Unsupported = ({
     }
 
     return (
-        <Timeline
-            className={classNames('manual-poi', {
-                'manual-poi--mobile': isMobile(),
-            })}
-            disabled_items={[2]}
-        >
-            <Timeline.Item item_title={localize('Please upload one of the following documents:')}>
-                <Documents documents={documents} toggleDetail={toggleDetail} />
-            </Timeline.Item>
-            <Timeline.Item item_title={localize('Upload your selfie')}>
```

Comment:
is there a requirement to remove `Upload your selfie` step?

Reply:
the step is still there, just according to new design without stepper we don’t indicate it on firs step